pkg/apis/build/v1alpha2/build_pod_test.go

```diff
@@ -718,13 +908,16 @@ func testBuildPod(t *testing.T, when spec.G, it spec.S) {
 				"home-dir",
 				"cache-dir",
 			}, names(pod.Spec.InitContainers[2].VolumeMounts))
+			tags := []string{}
+			for _, tag := range build.Spec.Tags {
+				tags = append(tags, "-tag="+tag)
+			}
```

Comment:
```suggestion
```
`-tag` is unsupported by the analyzer for older platform api.

Reply:
The code [here](https://github.com/pivotal/kpack/pull/889/files#diff-796b50408de7dbf79786503ffc9edfbc1eb29abc4c36902f37c6c5714606333bR233-R242) looks correct, so maybe it's at odds with this expectation.